tests/mock_server.py

```diff
@@ -1,20 +1,32 @@
 import asyncio
 import multiprocessing
+import socket
 
 # Third Party
 import aiohttp
 import aiohttp.web
 import pytest
 from aiohttp.web import StreamResponse
-
-# aiobotocore
-from tests.moto_server import MotoService, get_free_tcp_port, host
+from moto.server import ThreadedMotoServer
 
 _proxy_bypass = {
     "http": None,
     "https": None,
 }
 
+host = '127.0.0.1'
+
+
+def get_free_tcp_port(release_socket: bool = False):
+    sckt = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    sckt.bind((host, 0))
+    addr, port = sckt.getsockname()
+    if release_socket:
+        sckt.close()
```

Comment:
Amazon Q Developer pointed me to this issue: https://docs.aws.amazon.com/amazonq/detector-library/python/socket-close-platform-compatibility/

Reply:
we're not using socket file descriptors, we're using stream...and never going to run these tests on windows ;)